<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WkfBMPlotWidget</class>
 <widget class="QWidget" name="WkfBMPlotWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>515</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="UtQtGL2DPlotWidget" name="plot" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>300</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>600</width>
       <height>400</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="rightPanel" native="true">
     <layout class="QVBoxLayout" name="rightPanelLayout">
      <item>
       <widget class="QPushButton" name="initialCountButton">
        <property name="text">
         <string>Initial Count Options</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="initialCountList">
        <property name="title">
         <string>Initial Counts</string>
        </property>
        <layout class="QVBoxLayout" name="initialCountLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Initial Time:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="initialTime">
           <property name="maximum">
            <double>999999.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="ut::qt::AxisBoundsWidget" name="axisBounds">
        <property name="title">
         <string>GroupBox</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ut::qt::AxisBoundsWidget</class>
   <extends>QGroupBox</extends>
   <header>UtQtAxisBoundsWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UtQtGL2DPlotWidget</class>
   <extends>QWidget</extends>
   <header>UtQtGL2DPlotWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
